baggers.js

```diff
@@ -15847,393 +15833,4 @@ var data = {
-          tags: ['Scrum', 'Culture', 'Agile', 'REX'],
-          lang: ['fr'],
-        },
-        {
-          title: 'git-101',
-          abstract:
-            'Présentation des concepts de base de git, et de son utilisation en local, en remote, et en panic.',
-          tags: ['git'],
-          lang: ['fr'],
-        },
-      ],
-      cities: ['Lille'],
-      contacts: {
-        twitter: 'tiste',
-        mail: 'baptiste.lecocq@gmail.com',
-      },
-    },
-    {
-      since: '2019-02-12',
-      name: 'Hubert SABLONNIÈRE',
-      bio:
-        'Développeur Web chez Clever Cloud',
-      picture: 'https://www.hsablonniere.com/theme/img/hsablonniere.jpg',
-      websites: [
-        {
-          name: 'Web',
-          url: 'https://www.hsablonniere.com/',
-        },
-        {
-          name: 'Github',
-          url: 'https://github.com/hsablonniere',
-        },
-      ],
-      location: 'Lille',
-      sessions: [
-        {
-          title: '#BackToBasics : Les cookies HTTP',
-          abstract:
-            "Les cookies HTTP sont partout ! Pas une journée sans qu’un site nous demande “d’accepter les cookies”. Rares sont les projets Web qui ne les utilisent pas et pourtant leur fonctionnement nous échappe trop souvent.\n\nOn confond les règles auxquelles ils sont soumis au sein des navigateurs. On fait des amalgames avec les sessions côté serveur. Ces petits trous dans le gigantesque éventail de connaissances du développeur moderne ont parfois des impacts non négligeables sur la sécurité de nos applications et sur la préservation de notre vie privée en tant qu’utilisateur du Web.\n\nAu menu de cette session, je vous propose de (re)découvrir les cookies HTTP par l’exemple et la pratique. Nous reviendrons sur leurs origines et nous verrons en détails leur fonctionnement. Enfin, nous aborderons les aspects les plus récents dans le domaine (SameSite, cookie prefix...). Ce talk sera plus généralement l’occasion de parler de sécurité, de tracking et de vie privée.",
-          tags: ['Web', 'Sécurité', 'Cookies', 'Auth'],
-          lang: ['fr'],
-        },
-        {
-          title: 'D\'autres sujets ?',
-          abstract:
-            "Pourquoi pas, vous pouvez les retrouver ici et là :\n\nhttps://www.youtube.com/playlist?list=PLTBioAEvUEj0oFF2v71IPvu8iFBQYYNEt\nhttps://vimeo.com/album/4407825",
-          tags: [],
-          lang: ['fr', 'en'],
-        }
-      ],
-      cities: ['Lille', 'Paris', 'Nantes'],
-      contacts: {
-        twitter: 'hsablonniere',
-        mail: 'hubert.sablonniere+bbl@gmail.com',
-      },
-    },
-  ],
-  cities: {
```

Comment:
We do need cities...

Reply:
ok, so Github web editor is not the best tool for this :-\

Reply:
Yeah. I suspect this is because we have a toooooo big file here.